api/apps/api/config/custom-environment-variables.json

```diff
@@ -32,5 +32,19 @@
     "sharedFileStorage": {
       "localPath": "API_SHARED_FILE_STORAGE_LOCAL_PATH"
     }
+  },
+  "marxan": {
+    "inputFiles": {
+      "inputDat": {
+        "ioSettings": {
+          "INPUTDIR": "MARXAN_IO_SETTING_INPUTDIR",
+          "PUNAME": "MARXAN_IO_SETTING_PUNAME",
+          "SPECNAME": "MARXAN_IO_SETTING_SPECNAME",
+          "PUVSPRNAME": "MARXAN_IO_SETTING_PUVSPRNAME",
+          "BOUNDNAME": "MARXAN_IO_SETTING_BOUNDNAME",
+          "OUTPUTDIR": "MARXAN_IO_SETTING_OUTPUTDIR"
+        }
```

Comment:
INPUTDIR & OUTPUTDIR

not sure if those makes sense. Due to Marxan's CLI nature, we need to create directories it runs as temporary ones per run (and in fact, clean them up afterwards)

Reply:
other files names also may not have any point, as we will be creating them manually as well (and no way to get them afterwards - unless we want to keep them ofc)

Reply:
I've taken the list from the task description
@hotzevzl could you address these doubts?

Reply:
@kgajowy my plan was to create per-run root directories on fs and then use the same names for folders and files within each per-run root - so all these settings should be relative to each root. I would not use the marxan settings to point the CLI to the full path of input and output folders.
As for creating these config settings, they could even be hardcoded constants in the application source, but we will also in fact allow users to download the input files (https://vizzuality.atlassian.net/browse/MARXAN-308), so in case instance admins want to expose these files with non-standard names to users, then they could do so by fiddling with config rather than with the application source.

Reply:
@hotzevzl
ah, I missed they are relative. All fine!
Good luck, thats what I did in https://github.com/Vizzuality/marxan-cloud/pull/277 as well (per-run directories).